hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making_config_map.py

```diff
@@ -234,5 +236,22 @@ def update_oracle_settings(value: str):
         default=Decimal("5"),
         type_str="decimal",
         validator=lambda v: validate_decimal(v, Decimal(0), Decimal(100), inclusive=True)
+    ),
+    "debug_price_shim": ConfigVar(
+        key="debug_price_shim",
+        prompt="Do you want to enable the debug price shim for integration tests? If you don't know what this does "
+               "you should keep it disabled. >>> ",
+        default=False,
+        validator=validate_bool,
+        type_str="bool"
+    ),
+    "gateway_transaction_cancel_interval": ConfigVar(
+        key="gateway_transaction_cancel_interval",
+        prompt="After what time should blockchain transactions be cancelled if they are not included in a block? "
+               "(this only affects decentralized exchanges) (Enter time in seconds) >>> ",
+        prompt_on_new=True,
+        default=600,
+        validator=lambda v: validate_int(v, min_value=1, inclusive=True),
+        type_str="int"
```

Comment:
This two configuration are only required when the taker market is a gateway connector. We should add the parameter `required_if` to express the condition.

Reply:
✅